### ISSUE TITLE
Different evaluation results with the provided pretrained model

### ISSUE THREAD

**Issue by onedayinspring** [lzhyu/PoseShield#3]
Hi, your work is amazing! Thank you very much for releasing the code and pretrained models.

I followed the installation and evaluation instructions provided in the repository and evaluated the provided pretrained model using the default configuration. However, I obtained a different result from the one reported in the paper.

-----------------------------------------------------------------

The following is my eval result: 

Sample 500/500 status: solver_success=True, constraint_satisfied=True, exact_collision_free=True, final_cost=0.008689, constraint_value=0.100009, penetration_depth=0.000000, solver_message=Optimization terminated successfully

100%|███████████████████████████████████████████████████████████████████| 500/500 [**2:04:09<00:00**, 14.90s/it]

solver success rate 0.494

constraint satisfied rate 0.332

exact collision-free rate **0.836**

final error 0.024154225064563425

penetration depth reduction **0.9101637143971392**

average optimization time 7.7030 seconds

average MVD **0.039749**

-----------------------------------------------------------------
This should be the reported result
<img width="861" height="420" alt="Image" src="https://github.com/user-attachments/assets/ee61d896-1cfe-4da8-91fa-ec236e40a35b" />

-----------------------------------------------------------------

I used the provided pretrained checkpoint and default evaluation configuration, and verified that the model loads successfully and that the evaluation configuration matches the one provided in the repository.

-----------------------------------------------------------------

The following is my **args**: 

model_path=\'ckpts/poseshield/model.pth\', 

data_dir=\'data/dataset_test/\', 

n_samples=500, max_itr=300, lr=0.02, 

threshold=0.1, save=True, 

config_path=\'ckpts/poseshield/config.yaml\', 

cost_type=\'weighted\', 

tol=0.1)'

-----------------------------------------------------------------

The following is my **cfg**:

DATA: DIR: data/dataset/

MODEL: ACTIVATION: relu HIDDEN_DIM: 512 IN_DIM: 126 NUM_LAYERS: 12 TYPE: ResidualMLP

TRAIN: BATCH_SIZE: 256 DT: 0.01 EXP_NAME: train_active_large_v2 GRAD_LOSS_WEIGHT: 0.1 LR: 0.0002 NUM_EPOCHS: 300 TD_LOSS_WEIGHT: 1.0 VAL_INTERVAL: 10

-----------------------------------------------------------------

Could you please let me know if there are any additional details required to reproduce the reported results, such as a particular random seed, checkpoint version, evaluation split, or other evaluation settings that may not be documented in the repository?

Thank you very much for your help!
